test.sh

```diff
@@ -232,7 +233,10 @@ function nightly_functionality_stress_tests() {
 # of execution, especially to shake out AIO / thread registration issues.
 # #############################################################################
 function nightly_unit_stress_tests() {
-    local use_shmem=$1
+    local use_shmem=""
+    if [ $# -eq 1 ]; then
+        use_shmem="$1"
+   fi
```

Comment:
This is the pattern of the fix applied to multiple test-execution functions, to deal with the case when `test.sh` is called naming this test-execution function, with no further parameters.